users/views.py

```diff
@@ -0,0 +1,45 @@
+import json
+import bcrypt
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+from users.validator import *
```

Comment:
`*`로 모든 것을 다 불러오는 것은 최대한 삼가해주세요. 이후에 validator가 추가될 경우 불필요한 import가 될 수 있습니다.
```suggestion
from users.models    import User
from users.validator import email_validation, password_validation
```